src/Atmos/Model/source.jl

```diff
@@ -59,7 +59,12 @@ struct RayleighSponge{FT} <: Source
   zsponge::FT
   "Sponge Strength 0 ⩽ c_sponge ⩽ 1"
   c_sponge::FT
+  #"Reference relaxation velocity"
+  u_ref::FT
```

Comment:
```suggestion
 "Lateral reference velocity, u[m/s]"
  u_ref::FT
```
Likewise for `v`, `w`